local_planner/src/nodes/waypoint_generator.cpp

```diff
@@ -143,9 +142,8 @@ void WaypointGenerator::backOff() {
   output_.goto_position = toPoint(toEigen(pose_.pose.position) + dir);
   output_.goto_position.z = planner_info_.back_off_start_point.z;
 
-  output_.position_waypoint = createPoseMsg(output_.goto_position, last_yaw_);
+  output_.position_waypoint = createPoseMsg(output_.goto_position, curr_yaw_);
```

Comment:
This function is not really in use anymore, so I don't think it matters much. But I would say that last_yaw was a bit more correct, as it reduces drift in yaw orientation a bit